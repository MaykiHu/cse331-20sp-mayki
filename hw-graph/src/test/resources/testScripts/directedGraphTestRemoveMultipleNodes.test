#Create a graph
CreateGraph graph1

#Add a node to graph
AddNode graph1 n1

#Add a node to graph
AddNode graph1 n2

#Remove a pair of nodes from the graph
RemoveNode graph1 n1
RemoveNode graph1 n2