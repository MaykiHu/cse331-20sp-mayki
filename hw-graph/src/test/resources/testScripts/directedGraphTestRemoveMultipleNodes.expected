#Create a graph
created graph graph1

#Add a node to graph
added node n1 to graph1

#Add a node to graph
added node n2 to graph1

#Remove a pair of nodes from the graph
removed node n1 from graph1
removed node n2 from graph1